flink-python/pyflink/table/table_environment.py

```diff
@@ -153,9 +240,51 @@ def __init__(self, j_tenv):
         self._j_tenv = j_tenv
         super(StreamTableEnvironment, self).__init__(j_tenv)
 
+    def get_config(self):
+        """
+        Returns the table config to define the runtime behavior of the Table API.
+
+        :return: Current :class:`TableConfig`.
+        """
+        table_config = TableConfig()
+        table_config._j_table_config = self._j_tenv.getConfig()
+        table_config.is_stream = True
```

Comment:
Currently there are two ways to set the table config properties: in TableConfig and TableConfig.Builder. What about removing all the set methods in TableConfig?

Reply:
I think we'd better keep those set method but set them to private, because without them we need to put those setters' logic in TableConfig.Builder. I think it makes more sense to keep these logic in TableConfig.